controllers/board-game-review-controllers.js

```diff
@@ -24,3 +24,21 @@ exports.getReviews = (req, res, next) => {
     });
 };
 
+
+
+
+
+
+
+
+
+exports.getComments = (req, res, next) => {
+    const id = req.params.review_id;
+    Promise.all([checkIfReviewIdExists(id), isIdValid(id), selectComment(id)])
+    .then(([idExists, isValidId, comments]) => {
```

Comment:
This could have been handled as a PSQL error, specifically for isIdValid, but this logic works fine 👍